components/Header/Search/DesktopSearch.tsx

```diff
@@ -13,20 +13,25 @@ interface Props {
   setSearch: Dispatch<SetStateAction<Search>>;
   desktopInputRef: RefObject<HTMLInputElement>;
   filteredPosts: PostDocumentWithoutContent[];
+  setShowSearch: (ShowSearch: boolean) => void;
```

Comment:
@ntorbinskiy variables in JS if that's not the constant starts from the lowercase letter